lua/telescope/_extensions/file_browser.lua

```diff
@@ -173,6 +169,22 @@ local file_browser = function(opts)
   fb_picker(popts)
 end
 
+local fb_setup = function(opts)
+  -- TODO maybe merge other keys as well from telescope.config
+  pconf.mappings = vim.tbl_deep_extend("force", pconf.mappings, require("telescope.config").values.mappings)
+  pconf = vim.tbl_deep_extend("force", pconf, opts)
+
+  if pconf.hijack_netrw then
+    vim.cmd "silent! autocmd! FileExplorer *"
```

Comment:
- switch to new autocmd api
- `nvim lua/` in file browser should open `file browser` in that folder analogous to netrw
- Didn't we discuss to not open `file_browser` by default on `VimEnter`? It should properly follow netrw and only trigger file browser in `nvim .` where `.` indicates `$CWD`